### ISSUE TITLE
I need help with the rclone file 

### ISSUE THREAD

**Issue by Mrtkyy** [vKolerts/sidenoder#32]
Hi there, I recently updated with the latest sidenoder version (for mac) and immediately I lost the mount connection. As far as I understand I need to update the rclone file but I couldn't find the correct one. I'd appreciate any help.

<img width="308" alt="Ekran Resmi 2022-05-12 15 40 58" src="https://user-images.githubusercontent.com/105434800/168078785-7d37f938-eef8-4563-acb6-6fa3fd0864f7.png">
<img width="1128" alt="Ekran Resmi 2022-05-12 15 52 30" src="https://user-images.githubusercontent.com/105434800/168078979-66618150-fadf-4fa1-b346-25d3503f4817.png">

**Comment by Mrtkyy** [comment]
Yeah, I couldn't understand the real issue as well. I  just upgraded the latest sidenoder then it started to give errors about month disconnected. After a little search I though I needed to change the rclone file. 

I don't really know what is the main problem here but for example if I download the latest rclone file on the telegram (which is  named as : rclone_all_170222.conf or rclone_ffa_271221.conf)  I no longer receive the error message and I can see the quest games on the remote folder.

But whenever I try to download any game I receive this error instead:
<img width="1623" alt="Ekran Resmi 2022-05-12 17 36 14" src="https://user-images.githubusercontent.com/105434800/168101436-f64e849f-8a07-444a-935b-2b2aa3e543ab.png">
<img width="1529" alt="Ekran Resmi 2022-05-12 16 06 53" src="https://user-images.githubusercontent.com/105434800/168101475-8567bc8e-a172-4cc0-8e29-ecf2716fad39.png">

**Comment by KaladinDMP** [comment]
> Yeah, I couldn't understand the real issue as well. I just upgraded the latest sidenoder then it started to give errors about month disconnected. After a little search I though I needed to change the rclone file.
> 
> I don't really know what is the main problem here but for example if I download the latest rclone file on the telegram (which is named as : rclone_all_170222.conf or rclone_ffa_271221.conf) I no longer receive the error message and I can see the quest games on the remote folder.
> 
> But whenever I try to download any game I receive this error instead

Interesting. Are you seeing the games as a gamelist or like game cards with pictures? If it's a game list with no pictures that's usually a dead or over quota mirror but the normal error message would be I/o error. Have you tried changing mirrors? Try to use mirror 2,3,4 or 6 from FFA and see if you get the same error. 

Is this an error you get while browsing the temp directory created by the mount or is this while trying to install games? Because I do believe if this is the temp mounted folder they only have 0kb placeholders until the files themselves are actually downloaded during the install process.

As I said I use windows (also don't use Sidenoder as my primary install method) so I may be off here but I help troubleshoot this program quite often. If you continue to get no answer from the dev here, consider making a post on r/questpiracy subreddit. I'm a mod there and there's many users there that also use this program that may be able to help you.

**Comment by vKolerts** [comment]
FFA1 mirror has some empty folders. Try other mirrors as Kaladin recommend. I will update mirrors at few days. But I think, FFA1 will still has issues
